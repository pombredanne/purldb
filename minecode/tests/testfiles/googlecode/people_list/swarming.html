


<!DOCTYPE html>
<html>
<head>
 <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" >
 <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" >
 
 <meta name="ROBOTS" content="NOARCHIVE">
 
 <link rel="icon" type="image/vnd.microsoft.icon" href="https://ssl.gstatic.com/codesite/ph/images/phosting.ico">
 
 
 <script type="text/javascript">
 
 (function(){(function(){function c(a){this.t={};this.tick=function(a,b,c){b=void 0!=c?c:(new Date).getTime();this.t[a]=b;if(void 0==c)try{window.console.timeStamp("CSI/"+a)}catch(d){}};this.tick("start",null,a)}var a;window.performance&&(a=window.performance.timing);var b=a?new c(a.responseStart):new c;window.jstiming={Timer:c,load:b};a&&(b=a.navigationStart,a=a.responseStart,0<b&&a>=b&&(window.jstiming.srt=a-b));try{a=null,window.chrome&&window.chrome.csi&&(a=Math.floor(window.chrome.csi().pageT)),null==a&&
window.gtbExternal&&(a=window.gtbExternal.pageT()),null==a&&window.external&&(a=window.external.pageT),a&&(window.jstiming.pt=a)}catch(d){}})();})();

 
 
 
 
 var codesite_token = null;
 
 
 var CS_env = {"relativeBaseUrl": "", "loggedInUserEmail": null, "profileUrl": null, "domainName": null, "assetHostPath": "https://ssl.gstatic.com/codesite/ph", "projectName": "swarming", "assetVersionPath": "https://ssl.gstatic.com/codesite/ph/13484908847848575064", "projectHomeUrl": "/p/swarming", "token": null};
 var _gaq = _gaq || [];
 _gaq.push(
 ['siteTracker._setAccount', 'UA-18071-1'],
 ['siteTracker._trackPageview']);
 
 (function() {
 var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
 ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
 (document.getElementsByTagName('head')[0] || document.getElementsByTagName('body')[0]).appendChild(ga);
 })();
 
 </script>
 
 
 <title>People - 
 swarming -
 
 
 Massively parallel test execution - Google Project Hosting
 </title>
 <link type="text/css" rel="stylesheet" href="https://ssl.gstatic.com/codesite/ph/13484908847848575064/css/core.css">
 
 <link type="text/css" rel="stylesheet" href="https://ssl.gstatic.com/codesite/ph/13484908847848575064/css/ph_list.css" >
 
 
 
 
<!--[if IE]>
 <link type="text/css" rel="stylesheet" href="https://ssl.gstatic.com/codesite/ph/13484908847848575064/css/d_ie.css" >
<![endif]-->
 <style type="text/css">
 .menuIcon.off { background: no-repeat url(https://ssl.gstatic.com/codesite/ph/images/dropdown_sprite.gif) 0 -42px }
 .menuIcon.on { background: no-repeat url(https://ssl.gstatic.com/codesite/ph/images/dropdown_sprite.gif) 0 -28px }
 .menuIcon.down { background: no-repeat url(https://ssl.gstatic.com/codesite/ph/images/dropdown_sprite.gif) 0 0; }
 
 
 #resultstable td, #resultstable th { padding-right:2em }

 
 </style>
</head>
<body class="t1">
<script type="text/javascript">
 window.___gcfg = {lang: 'en'};
 (function() 
 {var po = document.createElement("script");
 po.type = "text/javascript"; po.async = true;po.src = "https://apis.google.com/js/plusone.js";
 var s = document.getElementsByTagName("script")[0];
 s.parentNode.insertBefore(po, s);
 })();
</script>
<div class="headbg">

 <div id="gaia">
 

 <span>
 
 
 <a href="#" id="projects-dropdown" onclick="return false;"><u>My favorites</u> <small>&#9660;</small></a>
 | <a href="https://www.google.com/accounts/ServiceLogin?service=code&amp;ltmpl=phosting&amp;continue=https%3A%2F%2Fcode.google.com%2Fp%2Fswarming%2Fpeople%2Flist&amp;followup=https%3A%2F%2Fcode.google.com%2Fp%2Fswarming%2Fpeople%2Flist" onclick="_CS_click('/gb/ph/signin');"><u>Sign in</u></a>
 
 </span>

 </div>

 <div class="gbh" style="left: 0pt;"></div>
 <div class="gbh" style="right: 0pt;"></div>
 
 
 <div style="height: 1px"></div>
<!--[if lte IE 7]>
<div style="text-align:center;">
Your version of Internet Explorer is not supported. Try a browser that
contributes to open source, such as <a href="http://www.firefox.com">Firefox</a>,
<a href="http://www.google.com/chrome">Google Chrome</a>, or
<a href="http://code.google.com/chrome/chromeframe/">Google Chrome Frame</a>.
</div>
<![endif]-->



 <table style="padding:0px; margin: 0px 0px 10px 0px; width:100%" cellpadding="0" cellspacing="0"
 itemscope itemtype="http://schema.org/CreativeWork">
 <tr style="height: 58px;">
 
 
 
 <td id="plogo">
 <link itemprop="url" href="/p/swarming">
 <a href="/p/swarming/">
 
 <img src="https://ssl.gstatic.com/codesite/ph/images/defaultlogo.png" alt="Logo" itemprop="image">
 
 </a>
 </td>
 
 <td style="padding-left: 0.5em">
 
 <div id="pname">
 <a href="/p/swarming/"><span itemprop="name">swarming</span></a>
 </div>
 
 <div id="psum">
 <a id="project_summary_link"
 href="/p/swarming/"><span itemprop="description">Massively parallel test execution</span></a>
 
 </div>
 
 
 </td>
 <td style="white-space:nowrap;text-align:right; vertical-align:bottom;">
 
 <form action="/hosting/search">
 <input size="30" name="q" value="" type="text">
 
 <input type="submit" name="projectsearch" value="Search projects" >
 </form>
 
 </tr>
 </table>

</div>

 
<div id="mt" class="gtb"> 
 <a href="/p/swarming/" class="tab active">Project&nbsp;Home</a>
 
 
 
 
 
 
 <a href="/p/swarming/w/list" class="tab ">Wiki</a>
 
 
 
 
 
 <a href="/p/swarming/issues/list"
 class="tab ">Issues</a>
 
 
 
 
 
 <a href="/p/swarming/source/checkout"
 class="tab ">Source</a>
 
 
 
 
 
 
 
 
 <div class=gtbc></div>
</div>
<table cellspacing="0" cellpadding="0" width="100%" align="center" border="0" class="st">
 <tr>
 
 
 
 
 
 
 
 <td class="subt">
 <div class="st3">
 <div class="isf">
 <span class="inst1">
 <a href="/p/swarming/">Summary</a>
 </span>
 
 
 &nbsp;
 <span class="inst3">
 <a href="/p/swarming/people/list">People</a>
 </span>
 
 
 </div>
</div>

 </td>
 
 
 <td align="right" valign="top" class="bevel-right"></td>
 </tr>
</table>


<script type="text/javascript">
 var cancelBubble = false;
 function _go(url) { document.location = url; }
</script>
<div id="maincol"
 
>

 





<script type="text/javascript">
function _showBelow(){}
function _toggleStar(){}
function _rowRolloverOn(){}
function _rowRolloverOff(){}
function _goIssue(){}
function _goFile(){}
</script>

<script type="text/javascript">
 var cancelBubble = false;
</script>
<form method="POST" action="addMembers.do"
 >
<div id="colcontrol">
 <div class="list">
 <div class="pagination">
 
 1 - 10 of 10
 
 </div>
 <b>Project People</b>
 &nbsp; &nbsp;
 
 </div>
 <table cellspacing="0" cellpadding="2" border="0" class="results" id="resultstable" width="100%">
 
 <thead>
 <tr id="headingrow">
 
 
 <th style="white-space:nowrap">Name</th>
 
 <th style="white-space:nowrap">Role</th>
 
 <th style="white-space:nowrap">Duties</th>
 
 <th style="white-space:nowrap">Notes</th>
 
 </tr>
 </thead>
 
 <tr onmouseover="_rowRolloverOn(this)" onmouseout="_rowRolloverOff(this); cancelBubble=false">
 
 
 
 
 
 <td class="vt col_0" onclick="if (!cancelBubble) _go('/p/swarming/people/detail?u=mad@chromium.org')" style="white-space:nowrap">
 <a onclick="cancelBubble=true" href="/p/swarming/people/detail?u=mad@chromium.org"
 >mad@chromium.org</a>
 
 </td>
 
 
 
 
 
 
 
 
 
 
 <td class="vt col_1" onclick="if (!cancelBubble) _go('/p/swarming/people/detail?u=mad@chromium.org')"
 ><a onclick="cancelBubble=true" href="/p/swarming/people/detail?u=mad@chromium.org"
 >
 
 
 Owner
 
 
 
 </a></td>
 
 
 
 
 
 
 
 
 <td class="vt col_2" onclick="if (!cancelBubble) _go('/p/swarming/people/detail?u=mad@chromium.org')" width="50%">
 
 <a onclick="cancelBubble=true" href="/p/swarming/people/detail?u=mad@chromium.org">----</a>
 
 </td>
 
 
 
 
 
 <td class="vt col_3" width="50%"
 onclick="if (!cancelBubble) _go('/p/swarming/people/detail?u=mad@chromium.org')"
 ><a onclick="cancelBubble=true" href="/p/swarming/people/detail?u=mad@chromium.org"
 >
 ----

</a>
 </td>
 
 
 
 
</tr>

 
 <tr onmouseover="_rowRolloverOn(this)" onmouseout="_rowRolloverOff(this); cancelBubble=false">
 
 
 
 
 
 <td class="vt col_0" onclick="if (!cancelBubble) _go('/p/swarming/people/detail?u=csharp@chromium.org')" style="white-space:nowrap">
 <a onclick="cancelBubble=true" href="/p/swarming/people/detail?u=csharp@chromium.org"
 >csharp@chromium.org</a>
 
 </td>
 
 
 
 
 
 
 
 
 
 
 <td class="vt col_1" onclick="if (!cancelBubble) _go('/p/swarming/people/detail?u=csharp@chromium.org')"
 ><a onclick="cancelBubble=true" href="/p/swarming/people/detail?u=csharp@chromium.org"
 >
 
 
 Owner
 
 
 
 </a></td>
 
 
 
 
 
 
 
 
 <td class="vt col_2" onclick="if (!cancelBubble) _go('/p/swarming/people/detail?u=csharp@chromium.org')" width="50%">
 
 <a onclick="cancelBubble=true" href="/p/swarming/people/detail?u=csharp@chromium.org">----</a>
 
 </td>
 
 
 
 
 
 <td class="vt col_3" width="50%"
 onclick="if (!cancelBubble) _go('/p/swarming/people/detail?u=csharp@chromium.org')"
 ><a onclick="cancelBubble=true" href="/p/swarming/people/detail?u=csharp@chromium.org"
 >
 ----

</a>
 </td>
 
 
 
 
</tr>

 
 <tr onmouseover="_rowRolloverOn(this)" onmouseout="_rowRolloverOff(this); cancelBubble=false">
 
 
 
 
 
 <td class="vt col_0" onclick="if (!cancelBubble) _go('/p/swarming/people/detail?u=maruel@chromium.org')" style="white-space:nowrap">
 <a onclick="cancelBubble=true" href="/p/swarming/people/detail?u=maruel@chromium.org"
 >maruel@chromium.org</a>
 
 </td>
 
 
 
 
 
 
 
 
 
 
 <td class="vt col_1" onclick="if (!cancelBubble) _go('/p/swarming/people/detail?u=maruel@chromium.org')"
 ><a onclick="cancelBubble=true" href="/p/swarming/people/detail?u=maruel@chromium.org"
 >
 
 
 Owner
 
 
 
 </a></td>
 
 
 
 
 
 
 
 
 <td class="vt col_2" onclick="if (!cancelBubble) _go('/p/swarming/people/detail?u=maruel@chromium.org')" width="50%">
 
 <a onclick="cancelBubble=true" href="/p/swarming/people/detail?u=maruel@chromium.org">----</a>
 
 </td>
 
 
 
 
 
 <td class="vt col_3" width="50%"
 onclick="if (!cancelBubble) _go('/p/swarming/people/detail?u=maruel@chromium.org')"
 ><a onclick="cancelBubble=true" href="/p/swarming/people/detail?u=maruel@chromium.org"
 >
 ----

</a>
 </td>
 
 
 
 
</tr>

 
 <tr onmouseover="_rowRolloverOn(this)" onmouseout="_rowRolloverOff(this); cancelBubble=false">
 
 
 
 
 
 <td class="vt col_0" onclick="if (!cancelBubble) _go('/p/swarming/people/detail?u=vadimsh@chromium.org')" style="white-space:nowrap">
 <a onclick="cancelBubble=true" href="/p/swarming/people/detail?u=vadimsh@chromium.org"
 >vadimsh@chromium.org</a>
 
 </td>
 
 
 
 
 
 
 
 
 
 
 <td class="vt col_1" onclick="if (!cancelBubble) _go('/p/swarming/people/detail?u=vadimsh@chromium.org')"
 ><a onclick="cancelBubble=true" href="/p/swarming/people/detail?u=vadimsh@chromium.org"
 >
 
 
 Owner
 
 
 
 </a></td>
 
 
 
 
 
 
 
 
 <td class="vt col_2" onclick="if (!cancelBubble) _go('/p/swarming/people/detail?u=vadimsh@chromium.org')" width="50%">
 
 <a onclick="cancelBubble=true" href="/p/swarming/people/detail?u=vadimsh@chromium.org">----</a>
 
 </td>
 
 
 
 
 
 <td class="vt col_3" width="50%"
 onclick="if (!cancelBubble) _go('/p/swarming/people/detail?u=vadimsh@chromium.org')"
 ><a onclick="cancelBubble=true" href="/p/swarming/people/detail?u=vadimsh@chromium.org"
 >
 ----

</a>
 </td>
 
 
 
 
</tr>

 
 <tr onmouseover="_rowRolloverOn(this)" onmouseout="_rowRolloverOff(this); cancelBubble=false">
 
 
 
 
 
 <td class="vt col_0" onclick="if (!cancelBubble) _go('/p/swarming/people/detail?u=108662825873451018571')" style="white-space:nowrap">
 <a onclick="cancelBubble=true" href="/p/swarming/people/detail?u=108662825873451018571"
 >ganet...@chromium.org</a>
 
 </td>
 
 
 
 
 
 
 
 
 
 
 <td class="vt col_1" onclick="if (!cancelBubble) _go('/p/swarming/people/detail?u=108662825873451018571')"
 ><a onclick="cancelBubble=true" href="/p/swarming/people/detail?u=108662825873451018571"
 >
 
 
 Committer
 
 
 
 </a></td>
 
 
 
 
 
 
 
 
 <td class="vt col_2" onclick="if (!cancelBubble) _go('/p/swarming/people/detail?u=108662825873451018571')" width="50%">
 
 <a onclick="cancelBubble=true" href="/p/swarming/people/detail?u=108662825873451018571">----</a>
 
 </td>
 
 
 
 
 
 <td class="vt col_3" width="50%"
 onclick="if (!cancelBubble) _go('/p/swarming/people/detail?u=108662825873451018571')"
 ><a onclick="cancelBubble=true" href="/p/swarming/people/detail?u=108662825873451018571"
 >
 ----

</a>
 </td>
 
 
 
 
</tr>

 
 <tr onmouseover="_rowRolloverOn(this)" onmouseout="_rowRolloverOff(this); cancelBubble=false">
 
 
 
 
 
 <td class="vt col_0" onclick="if (!cancelBubble) _go('/p/swarming/people/detail?u=kbr@chromium.org')" style="white-space:nowrap">
 <a onclick="cancelBubble=true" href="/p/swarming/people/detail?u=kbr@chromium.org"
 >kbr@chromium.org</a>
 
 </td>
 
 
 
 
 
 
 
 
 
 
 <td class="vt col_1" onclick="if (!cancelBubble) _go('/p/swarming/people/detail?u=kbr@chromium.org')"
 ><a onclick="cancelBubble=true" href="/p/swarming/people/detail?u=kbr@chromium.org"
 >
 
 
 Committer
 
 
 
 </a></td>
 
 
 
 
 
 
 
 
 <td class="vt col_2" onclick="if (!cancelBubble) _go('/p/swarming/people/detail?u=kbr@chromium.org')" width="50%">
 
 <a onclick="cancelBubble=true" href="/p/swarming/people/detail?u=kbr@chromium.org">----</a>
 
 </td>
 
 
 
 
 
 <td class="vt col_3" width="50%"
 onclick="if (!cancelBubble) _go('/p/swarming/people/detail?u=kbr@chromium.org')"
 ><a onclick="cancelBubble=true" href="/p/swarming/people/detail?u=kbr@chromium.org"
 >
 ----

</a>
 </td>
 
 
 
 
</tr>

 
 <tr onmouseover="_rowRolloverOn(this)" onmouseout="_rowRolloverOff(this); cancelBubble=false">
 
 
 
 
 
 <td class="vt col_0" onclick="if (!cancelBubble) _go('/p/swarming/people/detail?u=iannucci@chromium.org')" style="white-space:nowrap">
 <a onclick="cancelBubble=true" href="/p/swarming/people/detail?u=iannucci@chromium.org"
 >iannucci@chromium.org</a>
 
 </td>
 
 
 
 
 
 
 
 
 
 
 <td class="vt col_1" onclick="if (!cancelBubble) _go('/p/swarming/people/detail?u=iannucci@chromium.org')"
 ><a onclick="cancelBubble=true" href="/p/swarming/people/detail?u=iannucci@chromium.org"
 >
 
 
 Committer
 
 
 
 </a></td>
 
 
 
 
 
 
 
 
 <td class="vt col_2" onclick="if (!cancelBubble) _go('/p/swarming/people/detail?u=iannucci@chromium.org')" width="50%">
 
 <a onclick="cancelBubble=true" href="/p/swarming/people/detail?u=iannucci@chromium.org">----</a>
 
 </td>
 
 
 
 
 
 <td class="vt col_3" width="50%"
 onclick="if (!cancelBubble) _go('/p/swarming/people/detail?u=iannucci@chromium.org')"
 ><a onclick="cancelBubble=true" href="/p/swarming/people/detail?u=iannucci@chromium.org"
 >
 ----

</a>
 </td>
 
 
 
 
</tr>

 
 <tr onmouseover="_rowRolloverOn(this)" onmouseout="_rowRolloverOff(this); cancelBubble=false">
 
 
 
 
 
 <td class="vt col_0" onclick="if (!cancelBubble) _go('/p/swarming/people/detail?u=kmg@chromium.org')" style="white-space:nowrap">
 <a onclick="cancelBubble=true" href="/p/swarming/people/detail?u=kmg@chromium.org"
 >kmg@chromium.org</a>
 
 </td>
 
 
 
 
 
 
 
 
 
 
 <td class="vt col_1" onclick="if (!cancelBubble) _go('/p/swarming/people/detail?u=kmg@chromium.org')"
 ><a onclick="cancelBubble=true" href="/p/swarming/people/detail?u=kmg@chromium.org"
 >
 
 
 Committer
 
 
 
 </a></td>
 
 
 
 
 
 
 
 
 <td class="vt col_2" onclick="if (!cancelBubble) _go('/p/swarming/people/detail?u=kmg@chromium.org')" width="50%">
 
 <a onclick="cancelBubble=true" href="/p/swarming/people/detail?u=kmg@chromium.org">----</a>
 
 </td>
 
 
 
 
 
 <td class="vt col_3" width="50%"
 onclick="if (!cancelBubble) _go('/p/swarming/people/detail?u=kmg@chromium.org')"
 ><a onclick="cancelBubble=true" href="/p/swarming/people/detail?u=kmg@chromium.org"
 >
 ----

</a>
 </td>
 
 
 
 
</tr>

 
 <tr onmouseover="_rowRolloverOn(this)" onmouseout="_rowRolloverOff(this); cancelBubble=false">
 
 
 
 
 
 <td class="vt col_0" onclick="if (!cancelBubble) _go('/p/swarming/people/detail?u=vapier@chromium.org')" style="white-space:nowrap">
 <a onclick="cancelBubble=true" href="/p/swarming/people/detail?u=vapier@chromium.org"
 >vapier@chromium.org</a>
 
 </td>
 
 
 
 
 
 
 
 
 
 
 <td class="vt col_1" onclick="if (!cancelBubble) _go('/p/swarming/people/detail?u=vapier@chromium.org')"
 ><a onclick="cancelBubble=true" href="/p/swarming/people/detail?u=vapier@chromium.org"
 >
 
 
 Committer
 
 
 
 </a></td>
 
 
 
 
 
 
 
 
 <td class="vt col_2" onclick="if (!cancelBubble) _go('/p/swarming/people/detail?u=vapier@chromium.org')" width="50%">
 
 <a onclick="cancelBubble=true" href="/p/swarming/people/detail?u=vapier@chromium.org">----</a>
 
 </td>
 
 
 
 
 
 <td class="vt col_3" width="50%"
 onclick="if (!cancelBubble) _go('/p/swarming/people/detail?u=vapier@chromium.org')"
 ><a onclick="cancelBubble=true" href="/p/swarming/people/detail?u=vapier@chromium.org"
 >
 ----

</a>
 </td>
 
 
 
 
</tr>

 
 <tr onmouseover="_rowRolloverOn(this)" onmouseout="_rowRolloverOff(this); cancelBubble=false">
 
 
 
 
 
 <td class="vt col_0" onclick="if (!cancelBubble) _go('/p/swarming/people/detail?u=cmp@chromium.org')" style="white-space:nowrap">
 <a onclick="cancelBubble=true" href="/p/swarming/people/detail?u=cmp@chromium.org"
 >cmp@chromium.org</a>
 
 </td>
 
 
 
 
 
 
 
 
 
 
 <td class="vt col_1" onclick="if (!cancelBubble) _go('/p/swarming/people/detail?u=cmp@chromium.org')"
 ><a onclick="cancelBubble=true" href="/p/swarming/people/detail?u=cmp@chromium.org"
 >
 
 
 Contributor
 
 
 
 </a></td>
 
 
 
 
 
 
 
 
 <td class="vt col_2" onclick="if (!cancelBubble) _go('/p/swarming/people/detail?u=cmp@chromium.org')" width="50%">
 
 <a onclick="cancelBubble=true" href="/p/swarming/people/detail?u=cmp@chromium.org">----</a>
 
 </td>
 
 
 
 
 
 <td class="vt col_3" width="50%"
 onclick="if (!cancelBubble) _go('/p/swarming/people/detail?u=cmp@chromium.org')"
 ><a onclick="cancelBubble=true" href="/p/swarming/people/detail?u=cmp@chromium.org"
 >
 ----

</a>
 </td>
 
 
 
 
</tr>

 
 
 </table>
 <div class="list-foot">
 <div class="pagination">
 
 1 - 10 of 10
 
 </div>
 &nbsp;
 </div>
</div>




</form>

<script type="text/javascript" src="https://ssl.gstatic.com/codesite/ph/13484908847848575064/js/dit_scripts.js"></script>

 
 
 
 <script type="text/javascript" src="https://ssl.gstatic.com/codesite/ph/13484908847848575064/js/ph_core.js"></script>
 
 
 
 
</div> 

<div id="footer" dir="ltr">
 <div class="text">
 <a href="/projecthosting/terms.html">Terms</a> -
 <a href="http://www.google.com/privacy.html">Privacy</a> -
 <a href="/p/support/">Project Hosting Help</a>
 </div>
</div>
 <div class="hostedBy" style="margin-top: -20px;">
 <span style="vertical-align: top;">Powered by <a href="http://code.google.com/projecthosting/">Google Project Hosting</a></span>
 </div>

 
 


 
 
 <script type="text/javascript">_CS_reportToCsi();</script>
 
 </body>
</html>

