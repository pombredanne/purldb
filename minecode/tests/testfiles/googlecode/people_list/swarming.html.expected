([{'detail_url': u'http://code.google.com/u/mad@chromium.org',
   'person_id': u'mad@chromium.org',
   'role': u'owner',
   'username': u'mad@chromium.org'},
  {'detail_url': u'http://code.google.com/u/csharp@chromium.org',
   'person_id': u'csharp@chromium.org',
   'role': u'owner',
   'username': u'csharp@chromium.org'},
  {'detail_url': u'http://code.google.com/u/maruel@chromium.org',
   'person_id': u'maruel@chromium.org',
   'role': u'owner',
   'username': u'maruel@chromium.org'},
  {'detail_url': u'http://code.google.com/u/vadimsh@chromium.org',
   'person_id': u'vadimsh@chromium.org',
   'role': u'owner',
   'username': u'vadimsh@chromium.org'},
  {'detail_url': u'http://code.google.com/u/108662825873451018571',
   'person_id': u'108662825873451018571',
   'role': u'committer',
   'username': u'ganet...@chromium.org'},
  {'detail_url': u'http://code.google.com/u/kbr@chromium.org',
   'person_id': u'kbr@chromium.org',
   'role': u'committer',
   'username': u'kbr@chromium.org'},
  {'detail_url': u'http://code.google.com/u/iannucci@chromium.org',
   'person_id': u'iannucci@chromium.org',
   'role': u'committer',
   'username': u'iannucci@chromium.org'},
  {'detail_url': u'http://code.google.com/u/kmg@chromium.org',
   'person_id': u'kmg@chromium.org',
   'role': u'committer',
   'username': u'kmg@chromium.org'},
  {'detail_url': u'http://code.google.com/u/vapier@chromium.org',
   'person_id': u'vapier@chromium.org',
   'role': u'committer',
   'username': u'vapier@chromium.org'},
  {'detail_url': u'http://code.google.com/u/cmp@chromium.org',
   'person_id': u'cmp@chromium.org',
   'role': u'contributor',
   'username': u'cmp@chromium.org'}],
 [])